src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -42,14 +42,18 @@ public OI() {
      */
     public OI(Joystick driverStick, Joystick controlStick) {
         // Joystick for driving the robot around
+        
```

Comment:
We had controlStick down below to have a more readable flow of joystick commands, the driver ones go first and then below are the control ones, also your comments here are unaligned and there should be a space after the OI() contructor before you comment